library/Fission/Web.hs

```diff
@@ -33,37 +37,59 @@ import qualified Fission.Web.User    as User
 type API = Web.Swagger.API :<|> Web.API
 
 app ::
-  ( MonadDB               m
-  , MonadTime             m
-  , MonadLogger           m
-  , MonadDNSLink          m
-  , MonadLocalIPFS        m
-  , MonadRemoteIPFS       m
-  , MonadLinkedIPFS       m
-  , MonadReflectiveServer m
+  ( MonadReflectiveServer    m
+  , MonadLinkedIPFS          m
+  , MonadRemoteIPFS          m
+  , MonadLocalIPFS           m
+  , MonadDNSLink             m
+  , MonadLogger              m
+  , MonadLogger            t
+  , MonadThrow             t
+  , MonadTime                m
+  , MonadDB                t m
+  , Heroku.AddOn.Retriever t
+  , Heroku.AddOn.Destroyer t
+  , User.Creator           t
+  , User.Retriever         t
```

Comment:
These constraints are starting to get... long. Which isn't necessarily bad. But what would you think of making something like 
```haskell
type User.DB t = (User.Creator t, User.Retriever t, User.Destroyer t, User.Modifier t)
``` 
and then just using `User.DB t` in the constraint list?

The downside is we lose some specificity from the constraints

Reply:
Yeah, the downside is that we can easily get lazy and just slap `User.DB` on everything, negating many of the benefits. As long as we keep it as small as possible, I'm okay with this 👍

Reply:
Called it `User.CRUD` and `User.CID.CRUD`